packaging/deb/create-deb-pkg.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+cd "$(dirname "$0")"
+echo "cleanup"
+rm -rf ./build ./fastfetch fastfetch.deb
+
+echo "building fastfetch"
+mkdir build
+cd build
+cmake ../../..
+cmake --build . -j$(nproc)
```

Comment:
`cmake --build . --target fastfetch`. -j should be configured by the user with the MAKEFLAGS environment variable, not forced by default.

Reply:
Makes sense. Done